tools/go-cli/go-whisk-cli/commands/util.go

```diff
@@ -506,29 +508,23 @@ func unpackGzip(inpath string, outpath string) error {
     // Make sure the target file does not exist
     if _, err := os.Stat(outpath); err == nil {
         whisk.Debug(whisk.DbgError, "os.Stat reports file '%s' exists\n", outpath)
-        errStr := fmt.Sprintf(
-            wski18n.T("The file {{.name}} already exists.  Delete it and retry.",
-                map[string]interface{}{"name": outpath}))
+        errStr := fmt.Sprintf("The file %s already exists.  Delete it and retry.", outpath)
```

Comment:
@mdeuser, looks like all the externalization of strings in this file got replaced?